cg/services/bcl_convert_metrics_service/bcl_convert_metrics_service.py

```diff
@@ -111,3 +111,26 @@ def create_sample_run_metrics(
             base_mean_quality_score=mean_quality_score,
             created_at=datetime.now(),
         )
+
+    def create_sample_sequencing_metrics_for_flow_cell(
+        self,
+        flow_cell_directory: Path,
+        run_metrics_id: int,
```

Comment:
I do think we should pass the instrument_run here instead of just the id. I think the code should deal as little with the foreign keys as possible and instead work with the relationships

Reply:
Will address in next PR